docs/azurite-blob-emulator-windows.md

```diff
@@ -0,0 +1,47 @@
+### Setting up Azure Blob Store emulator locally
+
+To run a local emulator, there are a number of required steps. In order to successfully write to the emulator, it is necessary to configure it over https, using a self-signed certificate, and with a Shared Access Signature (SAS). Before you start, make sure you have followed the steps in [enabling-ad-locally](./enabling-ad-locally.md)
+
+1. The first step is to create a [local self-signed certificate](./self-signed-ssl.md), follow the instructions for windows.
+
+2. Once the certificate is created, it needs to be accessible by docker, in order to start the Azurite emulator with https. The following command will create a docker container running the emulator, mapping a local folder containing the certificate:
+
+Windows Command Line:
+
+```
+docker run -p 10000:10000 -p 10001:10001 -p 10002:10002 -v %cd%:/workspace --name pins_azurite -d mcr.microsoft.com/azure-storage/azurite azurite --blobHost 0.0.0.0  --queueHost 0.0.0.0 --tableHost 0.0.0.0 --cert /workspace/cert.pem --key /workspace/key.pem --oauth basic --skipApiVersionCheck
```

Comment:
This will run in the current directory, needs to ensure is where you have the certificates